releasenotes/notes/fix-course-export-b35c78a22be0c6b5.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Markdown content will now be exported between opening and
```

Comment:
Release notes should always be worded as if the release had already dropped, hence, "Markdown content is now exported" etc.